controller/array_action/array_mediator_svc.py

```diff
@@ -732,8 +732,8 @@ def _get_host_ports(self, host, attribute_name):
         ports = host.get(attribute_name, [])
         return ports if isinstance(ports, list) else [ports]
 
-    def get_host_by_host_identifiers(self, initiators):
-        logger.debug("Getting host name for initiators : {0}".format(initiators))
+    def _get_host_by_host_identifiers_slow(self, initiators):
+        logger.debug("Getting host name slowly for initiators : {0}".format(initiators))
```

Comment:
it sounds like we are laughing at ourselves..

Reply:
I just want an easy way to distinguish between the short and the long way

Reply:
I think the method name is enough.. (it will be shown in logs)

Reply:
what will be shown in the logs?

Reply:
the `slow` in the method name

Reply:
where? I don't see it in the logs (only the string sent to `logger.debug`)

Reply:
oh, I see it now

Reply:
done

Reply:
could you resolve this conversation?